### ISSUE TITLE
Update dependencies

### ISSUE THREAD

**Issue by gavin771** [macbre/phantomas#691]
When installing the module I get the following errors, is it possible to update these dependencies and/or suppress these errors
<img width="969" alt="screen shot 2017-05-01 at 12 44 39 pm" src="https://cloud.githubusercontent.com/assets/7811111/25589117/f6ed0656-2e70-11e7-9617-e10a6f23b190.png">

**Comment by macbre** [comment]
See #695 + #698

**Comment by macbre** [comment]
```
npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
```

is still reported, let's use [`uuid`](https://www.npmjs.com/package/uuid) as suggested

**Comment by macbre** [comment]
Fixed by #699
